### PR TITLE
Pin ipywidgets in tests

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

**Comment by trungleduc** [comment]
We need to revert this PR https://github.com/voila-dashboards/voila/pull/1193 to test against `ipywidgets` 7

**Comment by martinRenou** [comment]
Indeed that's it! Thanks.

File: tests/app/preheat_multiple_notebooks_test.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import os
 
 BASE_DIR = os.path.dirname(__file__)
-NOTEBOOK_EXECUTION_TIME = 3
+NOTEBOOK_EXECUTION_TIME = 2
```
Comment (trungleduc) [inline tests/app/preheat_multiple_notebooks_test.py:7]:
```suggestion
NOTEBOOK_EXECUTION_TIME = 3
```
It is not related to `ipywidgets`, I added it to increase CI robustness.

Comment (martinRenou) [reply tests/app/preheat_multiple_notebooks_test.py]: Thanks!

**Comment by martinRenou** [comment]
Replaced by #1217

**Pull request closed without merging**